services/collaboration/pkg/connector/httpadapter.go

```diff
@@ -51,9 +52,6 @@ func NewHttpAdapter(gwc gatewayv1beta1.GatewayAPIClient, cfg *config.Config) *Ht
 	}
 
 	httpAdapter.locks = &locks.NoopLockParser{}
-	if strings.ToLower(cfg.App.Name) == "microsoftofficeonline" {
-		httpAdapter.locks = &locks.LegacyLockParser{}
-	}
```

Comment:
What is the status? It seems we could remove the custom lock parsing... maybe add a TODO if it isn't the case?

Reply:
I will add a TODO